### DISCUSSION TITLE
9 Channel MIXED Switch Config Mode?

### DISCUSSION THREAD

**Discussion by Stardustsparklin** [ExpressLRS/ExpressLRS#1840] [Q&A]
Is it possible for ELRS to host a 9 Channel Full Resolution Mixed switch configuration mode?  The goal being to maximize the number of channels being sent at rate, while affording more available channels (at half rate).

Seems like sacrificing one full rate channel would afford space for two half rate channels (CRSF Ext Limit H).  But I'm a total newb so I could be way off.

The _Full Res 9 ch Mixed_ might look like:

1. **CRSF Ext Limits**
2. **CRSF Ext Limits**
3. **CRSF Ext Limits**
4. **CRSF Ext Limits**
5. **2-pos Arm**
6. **CRSF Ext Limits**
7. **CRSF Ext Limits**
8. **CRSF Ext Limits**
9. CRSF Ext Limits H (Half Rate)
10. CRSF Ext Limits H (Half Rate)

Perhaps even a _Full Res 10 or 11 ch Mixed_ mode as well?

Reason would be for fixed wing aircraft with flaps (ie full span), gear, bomb drop/speed break/cargo door, horizontal stabs, etc).  Thanks.

**Comment by JyeSmith** [comment]
The current V3 release does support up to 16 full channels.

Some detailed info here in the PR https://github.com/ExpressLRS/ExpressLRS/pull/1572

**Comment by Stardustsparklin** [comment]
Thanks Jye!  Sorry, my question was confusing.  I updated it to be more clear.  The 16 channel mode sends all channels at half rate.  I'm looking to
maximize at least 6 - 7 channels at full rate, while affording additional channels at half rate.

**Reply by JyeSmith** [reply]
On 900M half rate is still 50hz.  Any particular use case you need a faster rate?

**Comment by Stardustsparklin** [comment]
Another way to look at this request is:

### The Full Res 9 ch Mixed
1. **CRSF Ext Limits** [Elevator]
2. **CRSF Ext Limits** [Rudder]
3. **CRSF Ext Limits** [Aileron]
4. **CRSF Ext Limits** [Aileron] *For crow/reflex
5. **2-pos Arm**
6. **CRSF Ext Limits** [Throttle]
7. **CRSF Ext Limits** [Flap]
8. **CRSF Ext Limits** [Flap] *For full span ailerons
9. CRSF Ext Limits H (Half Rate) [Gear]
10. CRSF Ext Limits H (Half Rate) [Flight Mode]

Or

1. **CRSF Ext Limits** [Horizontal Stab]
2. **CRSF Ext Limits** [Horizontal Stab]
3. **CRSF Ext Limits** [Rudder]
4. **CRSF Ext Limits** [Throttle]
5. **2-pos Arm**
6. **CRSF Ext Limits** [Aileron]
7. **CRSF Ext Limits** [Aileron]
8. **CRSF Ext Limits** [Flap]
9. CRSF Ext Limits H (Half Rate) [Gear]
10. CRSF Ext Limits H (Half Rate) [Flight Mode]
